octopusdeploy/schema_project_group.go

```diff
@@ -59,12 +59,19 @@ func getProjectGroupDataSchema() map[string]*schema.Schema {
 
 func getProjectGroupSchema() map[string]*schema.Schema {
 	return map[string]*schema.Schema{
-		"description": getDescriptionSchema(),
-		"id":          getIDSchema(),
-		"name":        getNameSchema(true),
+		"description": {
```

Comment:
what if `getDescriptionSchema` took a parameter to use in the Description property, in this case you'd pass "project group"?